nucleof072rb/Core/Src/main.c

```diff
@@ -97,17 +92,20 @@ int main(void)
   MX_GPIO_Init();
   MX_USART2_UART_Init();
   MX_SPI1_Init();
-  MX_SPI2_Init();
   MX_TIM1_Init();
   /* USER CODE BEGIN 2 */
 
-    __HAL_RCC_GPIOB_CLK_ENABLE();
-  	GPIO_InitTypeDef GPIO_InitStruct = {0};
+  HAL_GPIO_WritePin(GPIOB, GPIO_PIN_8, GPIO_PIN_SET);
+  __HAL_TIM_SET_COMPARE(&htim1, TIM_CHANNEL_1, (0.05 * __HAL_TIM_GET_AUTORELOAD(&htim1)));
+  __HAL_RCC_GPIOB_CLK_ENABLE();
```

Comment:
Not 100% sure if I'm allowed to set the compare register before enabling the clock